### ISSUE TITLE
ptr.clone_checkout_strategy undefined when installing gopkg.in/libgit2/git2go.v26

### ISSUE THREAD

**Issue by rmerry** [libgit2/git2go#405]
Hey guys,

When trying to install gopkg.in/libgit2/git2go.v26 I get the following output:

```
richard:~/go/src/github.com  $ go get gopkg.in/libgit2/git2go.v26
# gopkg.in/libgit2/git2go.v26
../gopkg.in/libgit2/git2go.v26/submodule.go:372: ptr.clone_checkout_strategy undefined (type *C.struct_git_submodule_update_options has no field or method clone_checkout_strategy)
```

I have checkout out and installed v0.26 of libgit2. 

Interestingly the changelog for v0.26 of libgit2 says:

>Breaking API changes
>clone_checkout_strategy has been removed from git_submodule_update_option. The checkout strategy used to clone will be the same strategy specified in checkout_opts.

What am I doing wrong?

**Comment by reujab** [comment]
This is an issue with the bindings. The issue would be fixed if #402, #403, or #404 were merged, but the project hasn't been very active recently.

As a workaround until a pull requests is merged, you can use v25 or [reujab/git2go-v26](https://github.com/reujab/git2go-v26), which is forked from [mikolajb/git2go](https://github.com/mikolajb/git2go) and merged the v26 branch with master for use with `go get`.

**Comment by carlosmn** [comment]
Fixed via #402
